### ISSUE TITLE
Audit the code for all `NEEDS_LOC(x)` and fix them

### ISSUE THREAD

**Issue by DHowett-MSFT** [microsoft/terminal#4155]
This macro should be turned into a build break at our earliest convenience. If we ship with this macro in our code, we have failed.
